src/pages/addItemPage/components/TagEditor.jsx

```diff
@@ -0,0 +1,50 @@
+import XSvg from "@/assets/svg/XSvg";
+import Input from "@/components/common/Input";
+import { useState } from "react";
+import * as S from "./TagEditor.styles";
+
+const TagEditor = ({ tags = [], onAddTag, onRemoveTag }) => {
+  const [tagInput, setTagInput] = useState("");
+
+  const handleChange = (e) => {
+    setTagInput(e.target.value);
+  };
+
+  const handleKeyDown = (e) => {
+    if (e.key === "Enter") {
+      e.preventDefault();
+
+      const trimmed = tagInput.trim();
+      if (!trimmed) return;
+
+      if (onAddTag) {
+        onAddTag(trimmed);
+      }
+
+      setTagInput("");
```

Comment:
💊 제안
인풋의 값을 trim한 후 빈값이면 ealry return 하도록 작성해주셨어요. 이때 아래의 onAddTag 함수를 같이 확인해도 좋을 것 같아요.
```suggestion
      if (!trimmed || !onAddTag) return;

      onAddTag(trimmed);
      setTagInput("");
```